scripts/test

```diff
@@ -1,3 +1,4 @@
 #!/bin/bash
+# -p flag will create a different network for the test container so it won't interfere with the app container
```

Comment:
This comment doesn't match the description given in the docker compose documentation. See:

* https://docs.docker.com/compose/reference/overview/
* https://docs.docker.com/compose/reference/envvars/#compose-project-name